src/templates/angular/src/app/pages/header/header.component.ts

```diff
@@ -17,10 +18,10 @@ const $body = document.body; <%}%>
 export class HeaderComponent implements OnInit {
   <% if(isThemeProvider){%> isChecked: boolean = true;
   headerForm!: FormGroup;<%}%>
-  constructor( <% if(isThemeProvider){%> private fb: FormBuilder <%}%> <% if(isOkta && isThemeProvider) { %>,<% } %> <% if(isOkta) { %> private _router: Router, private _oktaStateService: OktaAuthStateService, @Inject(OKTA_AUTH) private _oktaAuth: OktaAuth <% } %>)  { }
+  constructor( <% if(isThemeProvider){%> private fb: FormBuilder <%}%> <% if(isOkta && isThemeProvider){ %>,<% } %> <% if(isOkta){ %>private _router: Router, private _oktaStateService: OktaAuthStateService, @Inject(OKTA_AUTH) private _oktaAuth: OktaAuth <%}%> <% if(isAuth0){ %>,public auth: AuthService<% } %>)  { }
   <% if(isOkta) { %>public name$!: Observable<string>;
   public isAuthenticated$!: Observable<boolean>;<% } %>
-
+  <% if(isAuth0){ %> logoutURL = 'http://localhost:4200/'; <% } %>
```

Comment:
let's get this URL (logoutURL/ appURL etc) from env, constant, config file
 http://localhost:4200/